From cd26850713088942ca4f9a248a8bed1f0504a58f Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 19 Dec 2024 15:11:40 +0000
Subject: [PATCH] fixup! misc: Add RP1 PIO driver

Change the Kconfig dependencies so that RP1_PIO depends on FIRMWARE_RP1,
rather than selecting it.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/misc/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/misc/Kconfig
+++ b/drivers/misc/Kconfig
@@ -19,7 +19,7 @@ config BCM2835_SMI
 
 config RP1_PIO
 	tristate "Raspberry Pi RP1 PIO driver"
-	select FIRMWARE_RP1
+	depends on FIRMWARE_RP1 || COMPILE_TEST
 	default n
 	help
 		Driver providing control of the Raspberry Pi PIO block, as found in
